docs/src/css/codes.css

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+.codes .name {
+  cursor: alias;
+}
+
+.codes .name:hover {
+  background: var(--ifm-font-color-base-inverse);
+}
+
+.codes .name:active {
+  color: var(--ifm-font-color-base-inverse);
+  background: var(--ifm-font-color-base);
+}
+
+.codes table {
+  display: table;
+  font-size: 0.8em;
+  margin-bottom: 0;
+}
+
+.codes th,
+.codes td {
+  padding: 0.2rem;
+}
+
+.codes td {
+  position: relative;
+}
+
+.codes .behavior {
+  width: 1%;
+  min-width: 1rem;
+  cursor: help;
+  color: var(--ifm-font-color-base);
+}
+
+.codes th.names,
+.codes th.description {
+  text-align: left;
+}
+
+.codes td.names code {
+  display: block;
+  float: left;
+  clear: both;
+  margin-top: 1px;
+  margin-bottom: 1px;
+  font-size: 0.85em;
+}
+
+.codes .context {
+  display: inline;
+  margin: 0 0 0 0.5em;
+  font-size: 0.85em;
+}
+
+.codes .context::before {
+  content: "(";
+}
+
+.codes .context::after {
+  content: ")";
+}
+
+.codes .symbol {
+  display: inline-flex;
+  flex-direction: column;
+}
+
+.codes .symbol code {
+  align-self: flex-start;
+  width: auto;
+}
+
+.codes .symbol .meaning {
+  flex: 1;
+  font-size: 0.8em;
+  text-transform: uppercase;
+}
+
+.codes .documentation,
+.codes .os {
+  width: 1%;
+  min-width: 2rem;
+  text-align: center;
+}
+
+.codes td.documentation {
+  font-size: 0.8em;
+}
+
+.codes td.documentation a {
+  cursor: help;
+}
+
+.codes .os {
+  color: black;
+}
+
+.codes td.os {
+  font-size: 0.9em;
+}
+
+.codes .not-tested {
+  font-weight: bold;
+}
+
+.codes .os.windows {
+  background: #81bdea;
+}
+
+.codes .os.linux {
+  background: #fde291;
+}
+
+.codes .os.android {
+  background: #77e6a9;
+}
```

Comment:
Small aesthetic thing. I'd prefer if these colors were a bit more pastel and subtle. They _feel_ out of place.

Reply:
Agreed, I didn't spend much time on that.  I welcome color suggestions!

Reply:
I think I've found some values I like:

- Windows: #CAEDFD
- Linux: #FFF2CA
- Android: #D8EED9

And then I'm thinking iOS could just be white and MacOS could take iOS's current color.

Result:
![image](https://user-images.githubusercontent.com/9439650/97750694-4bef9400-1abf-11eb-9bd1-29260aeadc2b.png)

We also may be able to add `mix-blend-mode: multiply` to `.os`. It keeps the every other row difference in brightness, but it looks terrible in dark mode. If we could turn it off in dark mode, it would be an improvement:
![image](https://user-images.githubusercontent.com/9439650/97751289-0d0e0e00-1ac0-11eb-9794-a1de538b6df8.png)

Reply:
Yeah, I like that.  Presumably it works well enough in dark mode also?

Reply:
Yeah, I like that!  Thanks for looking into it!  I'll give it a go :)

Reply:
Done.  Any further thoughts?